types/setting.go

```diff
@@ -920,6 +935,10 @@ func ValidateSetting(name, value string) (err error) {
 		if value != "true" && value != "false" {
 			return fmt.Errorf("value %v of setting %v should be true or false", value, sName)
 		}
+	case SettingNameDeletingConfirmationFlag:
+		if value != "true" && value != "false" {
+			return fmt.Errorf("value %v of setting %v should be true or false", value, sName)
+		}
```

Comment:
The implementation is the same as other bool setting, so why not just add this case before `SettingNameUpgradeChecker` then `fallthrough`?

Reply:
Ah. I see. Thank you. Fixed